src/components/userFeedback/UserFeedback.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { extend } from 'underscore';
+
+export interface IUserFeedbackProps {
+  feedbackText: string;
+  state: string;
+  extraClasses?: string;
+  displayOnShow?: string;
```

Comment:
I disagree. The displayOnShow is the display you want when the UserFeedback is visible, not always, whereas the displayClass is the final class that is added to the html element (which can be 'hidden' if the feedback state is VALID, or the value set on displayOnShow if the feedback state is WARNING or ERROR.